src/app/components/warning-label.tsx

```diff
@@ -12,7 +12,7 @@ export function WarningLabel({ children, title, ...props }: WarningLabelProps) {
     <Box bg="warning.background" borderRadius="sm" {...props}>
       <Flag
         align="top"
-        color="accent.notification-text"
+        color="ink.notification-text"
```

Comment:
It's throwing a type errors for `token('colors.warning.label') }}`

Reply:
I think maybe I broke types when I removed exporting `colors` with `tokens` from the mono repo?

Reply:
Nvm, I see I had a bunch of broken colors to still address. Types not broken.